conf.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+#
+# on_rtd is whether we are on readthedocs.org
+import os
+on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
+
+if not on_rtd:  # only import and set the theme if we're building docs locally
+    import sphinx_rtd_theme
+    html_theme = 'sphinx_rtd_theme'
+    html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+
+# otherwise, readthedocs.org uses their theme by default, so no need to specify it
+
+project = 'Fiware-Cloto'
```

Comment:
This file should have a newline at end of file according to pep8.
Please, add it before merge.